app.py

```diff
@@ -18,31 +18,43 @@
 
 def get_layout():
     boroughs = mapping.borough_area()
+    inventory_df = read_data.get_rental_inventory()
     inventory_graph = [
         dcc.Dropdown(sorted(boroughs.keys()),
                      placeholder='Select a borough', id='borough-dropdown'),
         dcc.Dropdown(id='areas-dropdown', placeholder='Areas'),
-        dcc.Graph(id='area-inventory')
+        dcc.Graph(id='area-inventory-line')
+    ]
+    inventory_sunburst = [
+        dcc.Dropdown(sorted(list(set(inventory_df['date'].dt.year))), id='year-dropdown',
```

Comment:
Calling `sorted` on an iterable in python will return a list.  So, casting your set to a list here is redundant.